tests/db/SelectTest.php

```diff
@@ -68,21 +81,95 @@ public function testBasicQueries()
 
         $this->assertEquals(
             [['now'=>4]],
-            $this->q()->field(new Expression('2+2'),'now')->get()
+            $this->q()->field(new Expression('2+2'), 'now')->get()
         );
 
         $this->assertEquals(
             [['now'=>6]],
-            $this->q()->field(new Expression('[]+[]',[3,3]),'now')->get()
+            $this->q()->field(new Expression('[]+[]', [3,3]), 'now')->get()
         );
     }
 
-    public function testExpresison()
+    public function testExpression()
     {
         $this->assertEquals(
             'foo',
-            $this->e('select []',['foo'])->getOne()
+            $this->e('select []', ['foo'])->getOne()
         );
     }
-}
 
+    /**
+     * WARNING: SQLite doesn't support TRUNCATE TABLE and can only use DELETE FROM instead
+     * so we disable this test while there is no solution how to run this test only on MySQL.
+     */
+    /*
+    public function testTruncate()
+    {
+        $this->q('employee')->truncate();
+        $this->assertEquals(
+            0,
+            $this->q('employee')->field(new Expression('count(*)'))->getOne()
+        );
+    }
+    */
+
+    public function testOtherQueries()
```

Comment:
Technically this needs to be in other db/* file because this one is called testSelectQueries, or we can just rename it.